.github/sync-repo-settings.yaml

```
@@ -0,0 +1,42 @@
+# Rules for main branch protection
+branchProtectionRules:
+# Identifies the protection rule pattern. Name of the branch to be protected.
+# Defaults to `main`
+- pattern: main
+  # Can admins overwrite branch protection.
+  # Defaults to `true`
+  isAdminEnforced: true
+  # Number of approving reviews required to update matching branches.
+  # Defaults to `1`
+  requiredApprovingReviewCount: 1
+  # Are reviews from code owners required to update matching branches.
+  # Defaults to `false`
+  requiresCodeOwnerReviews: true
+  # Require up to date branches
+  requiresStrictStatusChecks: false
```

```suggestion
  requiresStrictStatusChecks: true
```
I'd recommend to keep the default of true and follow up with a separate PR if we want to set this to false.
https://github.com/googleapis/repo-automation-bots/blob/4a530825cd1886bd583e602742fc55c03415505d/packages/sync-repo-settings/src/required-checks.json#L43